tests/tests.h

```diff
@@ -230,10 +236,10 @@ bool sirtest_os_log(void);
 bool sirtest_filesystem(void);
 
 /**
- * @test Ensure an empty message is properly handled.
+ * @ test
  * @returns bool `true` if the test passed, `false` otherwise.
```

Comment:
![IMG_7859](https://github.com/aremmell/libsir/assets/61629094/2f63f8ba-6c58-4a6a-ae7f-35cf8951c441)

You'll want to add a test description here, I think.

Reply:
@aremmell Also, need to run autoformat.sh.  Then GTG